src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -159,6 +172,7 @@ protected virtual Task<TResponse> HandleDelegatedResponseAsync(TResponse delegat
         TResponse? delegatedRequest;
         try
         {
+            using var _ = _telemetryReporter?.TrackLspRequest(nameof(HandleRequestAsync), LspName, positionInfo.LanguageKind == RazorLanguageKind.CSharp ? "Razor C# Language Server Client" : "HtmlDelegationLanguageServerClient", correlationId);
```

Comment:
TODO, move strings "Razor C# Language Server Client" and "HtmlDelegationLanguageServerClient" elsewhere

Reply:
They already are: https://github.com/dotnet/razor/blob/c3de7648d08114fcc3e77dac609afc77d197fbd2/src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPConstants.cs#L14

Reply:
RazorLSPConstants seemed inaccessible here. So I figured I had to create a new class

Reply:
oh, because it's LanguageServerClient. I wonder if we should move that?

Reply:
I think this call is just in the wrong place. The thing that receives this request is not the delegated servers, but our code in the LSP client. This call should love to DefauktRazorLanguageServerCustomMessageTarget, where the calls to the delegated servers happen, and then those constants will be available (and probably unnecessary anyway)